Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/ChangeEvent.swift

```diff
@@ -2,13 +2,22 @@ import MongoSwift
 
 /// The type of operation that occurred within a ChangeEvent.
 public enum OperationType: String, Codable {
-    case insert, delete, replace, update, unknown
+    case insert = "INSERT"
```

Comment:
[minor] I think they should actually be lowercase

Reply:
Are we sure?

Reply:
https://github.com/mongodb/stitch-android-sdk/blob/master/core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/ChangeEvent.java#L110